### PR TITLE
Handle variadic helpers

### PR DESCRIPTION
**Pull request by stanislas-m**
Allows helpers to receive an undetermined number of args.
It will be useful to improve helpers like `t` (i18n) which allows this kind of parameters.

### CODE REVIEW

File: plush_test.go [hunk 1]
```diff
@@ -342,7 +342,32 @@ func Test_UndefinedArg(t *testing.T) {
 
 	_, err := Render(input, ctx)
 	r.Error(err)
-	r.Equal(ErrUnknownIdentifier, errors.Cause(err))
```
Comment (markbates) [inline plush_test.go]: why did you remove this line in the test? does this PR change that unknown identifier functionality?

Comment (stanislas-m) [reply plush_test.go]: That's a typo... Thanks!

**Comment by stanislas-m** [comment]
@markbates Anything else on this one?

**Comment by markbates** [comment]
Sorry I didn't get to this sooner, was traveling and launching www.gopherguides.com yesterday, it was pretty crazy! 

This is awesome. Keep up the great work! Love it!

**Comment by stanislas-m** [comment]
Ah ah, no problem! It's good to take some time for yourself too. :)
